isis/src/galileo/apps/gllssi2isis/main.cpp

```diff
@@ -110,22 +109,16 @@ void IsisMain() {
     p.StartProcess();
   }
   else {
-    summedOutput = new Cube();
-    summedOutput->setDimensions(p.Samples() / 2, p.Lines() / 2, p.Bands());
-    summedOutput->setPixelType(p.PixelType());
-    summedOutput->create(ui.GetFileName("TO"));
-    p.StartProcess(translateData);
+    Isis::CubeAttributeOutput &att = Application::GetUserInterface().GetOutputAttribute("TO");
+    att.setPixelType(p.PixelType());
+    summedOutput = p.SetOutputCube("TO", att, p.Samples() / 2, p.Lines() / 2, p.Bands());
     ocube = summedOutput;
+    p.StartProcess(translateData);
   }
 
   translateLabels(pdsLabel, ocube);
   p.EndProcess();
 
-  if (summed) {
-    summedOutput->close();
-    delete summedOutput;
-  }
```

Comment:
Do we not still need to delete the summedOutput?

Reply:
This is handled by the ClearOutputCubes method in the Process class. Once EndProcess is called if you follow the call stack, all the cubes are closed and deleted.

Reply:
:+1: